RecoHGCal/TICL/plugins/TrackstersFromSimClustersProducer.cc

```diff
@@ -0,0 +1,151 @@
+// Author: Felice Pantaleo - felice.pantaleo@cern.ch
+// Date: 02/2021
+
+// user include files
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "DataFormats/CaloRecHit/interface/CaloCluster.h"
+#include "DataFormats/ParticleFlowReco/interface/PFCluster.h"
+
+#include "DataFormats/HGCalReco/interface/Trackster.h"
+
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "SimDataFormats/Associations/interface/LayerClusterToSimClusterAssociator.h"
+#include "SimDataFormats/CaloAnalysis/interface/SimCluster.h"
```

Comment:
shouldn't this be in the buildfile? Will this be the only dependency on Sim-related dependency in the RecoHGCAL/TICL package?

Reply:
@jpata we are not using anything from the library, why should it be needed in the `BuildFile.xml`?
This will be the only sim-related dependency in the package

Reply:
How does this code not need the library that defines SimCluster?
(regardless, things you depend on should be in the buildfile).

Reply:
I did it for the entire plugin folder, as this was the first time this request was made